.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: trusty
 language: node_js
 
 node_js:
-  - '6.9.1'
+  - '6.11.0'
```

Comment:
Have we decided that we are ready to move the node version up on our frost repos? If so, I will add it to the list of work to be done so that this change is consistently made across all of the frost repos.